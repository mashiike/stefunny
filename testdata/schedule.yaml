required_version: ">v0.0.0"

state_machine:
  name: Scheduled
  definition: hello_world.asl.json
  role_arn: arn:aws:iam::012345678901:role/service-role/StepFunctions-Hello-role
  logging_configuration:
    level: ALL
    destinations:
      - cloudwatch_logs_log_group:
          log_group_arn: arn:aws:logs:us-east-1:012345678901:log-group:/steps/hello


trigger:
  event:
    - rule_name: Scheduled-hourly
      schedule_expression: rate(1 hour)
      role_arn: arn:aws:iam::012345678901:role/service-role/Eventbridge-Hello-role
