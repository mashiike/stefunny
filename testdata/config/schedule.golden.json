{
  "required_version": "\u003ev0.0.0",
  "aws_region": "us-east-1",
  "state_machine": {
    "definition": "{\n   \"Comment\": \"A Hello World example of the Amazon States Language using Pass states\",\n   \"StartAt\": \"Hello\",\n   \"States\": {\n      \"Hello\": {\n         \"Next\": \"World\",\n         \"Type\": \"Pass\"\n      },\n      \"World\": {\n         \"End\": true,\n         \"Result\": \"World\",\n         \"Type\": \"Pass\"\n      }\n   }\n}",
    "logging_configuration": {
      "destinations": [
        {
          "cloudwatch_logs_log_group": {
            "log_group_arn": "arn:aws:logs:us-east-1:012345678901:log-group:/steps/hello"
          }
        }
      ],
      "level": "ALL"
    },
    "name": "Scheduled",
    "role_arn": "arn:aws:iam::012345678901:role/service-role/StepFunctions-Hello-role",
    "tracing_configuration": {},
    "type": "STANDARD"
  },
  "trigger": {
    "schedule": [
      {
        "name": "Scheduled-hourly",
        "schedule_expression": "rate(1 hour)",
        "schedule_expression_timezone": "Asia/Tokyo",
        "target": {
          "role_arn": "arn:aws:iam::012345678901:role/service-role/Eventbridge-Hello-role"
        }
      }
    ]
  },
  "ConfigDir": "testdata",
  "ConfigFileName": "schedule.yaml"
}